packages/sdk-v2/src/sdk/goodid/hooks/useCertificates.ts

```diff
@@ -0,0 +1,32 @@
+import { useCallback } from "react";
+import type { Certificate } from "../types";
+
+// This expects a single issuer (gd) for the mvp purpose
+export const useCertificates = (db: any) => {
+  const getCertificates: () => Promise<Certificate[] | undefined> = useCallback(async () => {
+    return await db.transaction("r", [db.certificates_credentials, db.certificates], async () => {
+      const certificateId = await db.certificates.primaryKeys();
+      const credentialsList = await db.certificates_credentials.where({ certificates_id: certificateId }).toArray();
+      return credentialsList;
```

Comment:
that doesnt seem right.
see how we add items elsewhere were we use dexie

Reply:
When doing multiple operations its what they say is best practice: https://dexie.org/docs/Tutorial/Best-Practices#5-use-transaction-scopes-whenever-you-plan-to-make-more-than-one-operation

Reply:
please do it like we do elsewhere. also the primaryKeys doesnt seem right